docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Density
+
+You can change the density of the rows and the column header.
+
+### Density prop
+
+The vertical density of the data grid can be set using the `density` prop. Regardless of its setting, the `density` prop will respect values provided using the `rowHeight` or `headerHeight` props.
```

Comment:
```suggestion
The vertical density of the data grid can be set using the `density` prop. The `density` prop applies the values determined by the `rowHeight` and `headerHeight` props, if supplied.  The user can override this setting with the toolbar density selector, if provided. 
```

(Hopefully I got the meaning correct this time,  now that I better understand it. Comment if it still isn't correct, or isn't clear. 🙂)